docs/content/troubleshooting/troubleshoot-support.md

```diff
@@ -0,0 +1,45 @@
+---
+docs: DOCS-1648
+doctypes:
+- troubleshooting
+draft: false
+tags:
+- docs
+title: Support
+toc: true
+weight: 100
+---
+
+F5 NGINX Ingress Controller adheres to the support policy detailed in the following knowledge base article: [K000140156](https://my.f5.com/manage/s/article/K000140156).
+
+In order to open a support ticket, F5 would like additional information to better understand the problem.
+
+The [nginx-supportpkg-for-k8s](https://github.com/nginxinc/nginx-supportpkg-for-k8s) plugin collects the information needed by F5 Technical Support to assist with troubleshooting your issue.
+
+The plugin leverages [krew](https://krew.sigs.k8s.io), the plugin manager for the Kubernetes [kubectl](https://kubernetes.io/docs/reference/kubectl/) command-line tool.
+
+The plugin may collect some or all of the following global and namespace-specific information:
+
+* k8s version, nodes information, and CRDs
```

Comment:
```suggestion
* K8s version, nodes information, and CRDs
```

Reply:
Thanks, previous commits have outdated your edit preventing me from committing it as is. 
I have adopted you suggestion as a separate commit.